usecases/schema/fakes_for_test.go

```diff
@@ -0,0 +1,142 @@
+package schema
```

Comment:
I would suggest renaming the file from fakes_for_test.go to fakes_test.go since it is implied by the context that mocks are intended for testing